### PR TITLE
Add solution to Leetcode 217

### PR DESCRIPTION
**Pull request by pandaxtc**
Add a solution class and 3 test cases for Leetcode 217, "Contains Duplicate"

### CODE REVIEW

File: LeetCode/217.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                return True
+            else:
+                seen.add(num)
+        return False
+        
+s = Solution()
+
+assert s.containsDuplicate([1,2,3,1]) == True
+assert s.containsDuplicate([1,2,3,4]) == False
+assert s.containsDuplicate([1,1,1,3,3,4,3,2,4,2]) == True
```
Comment (adityaarakeri) [inline LeetCode/217.py]: Could you be able to move the tests to the root `test folder? much appreciated
you can name your test as `test_217.py` and look into existing tests for implementaion
